content/influxdb/cloud-dedicated/get-started/query.md

```diff
@@ -338,36 +338,75 @@ _If your project's virtual environment is already running, skip to step 3._
     print(reader.to_pandas().to_markdown())
     ```
 
-    The sample code does the following:
+{{< expand-wrapper >}}
+{{% expand "<span class='req'>Important</span>: If using **Windows**, specify the **Windows** certificate path" %}}
 
-    1.  Imports the `InfluxDBClient3` constructor from the `influxdb_client_3` module.
-    
-    2.  Calls the `InfluxDBClient3()` constructor method with credentials to instantiate an InfluxDB `client` with the following credentials:
-
-        - **host**: {{% cloud-name %}} cluster URL (without `https://` protocol or trailing slash)
-        - **token**: a [database token](/influxdb/cloud-dedicated/admin/tokens/) with
-          read access to the specified database.
-          _For security reasons, we recommend setting this as an environment
-          variable rather than including the raw token string._
-        - **database**: the name of the {{% cloud-name %}} database to query
-    
-    3.  Defines the SQL query to execute and assigns it to a `query` variable.
+  If using a non-POSIX-compliant operating system (such as Windows), specify the root certificate path when instantiating the client.
+  The following example shows how to use the Python `certifi` package and client library options to pass the certificate path:
+
+  1.  In your terminal, install the Python `certifi` package.
+
+      ```sh
+      pip install certifi
+      ```
+  
+  2.  In your Python code, import `certifi` and call the `certifi.where()` method to retrieve the root certificate path.
+  3.  When instantiating the client, pass the `flight_client_options.tls_root_certs=<ROOT_CERT_PATH>` option with the certificate path--for example:
+
+      ```py
+      from influxdb_client_3 import InfluxDBClient3, flight_client_options
+      import os
+      import certifi
+
+      TOKEN = os.getenv('INFLUX_TOKEN')
+
+      fh = open(certifi.where(), "r")
+      cert = fh.read()
```

Comment:
Do they need to specify the path to read the cert from, or does this just work?

Reply:
You made me go re-read what Certify actually does 👍 I'll fix it.

> Certifi provides Mozilla’s carefully curated collection of Root Certificates for validating the trustworthiness of SSL certificates while verifying the identity of TLS hosts.